components/Main/ContactUs/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react'
+import MessageForm from './MessageForm'
+import style from './ContactUs.module.css'
+
+export default function ContactUs(): JSX.Element {
+  const [isSuccessful, setIsSuccessful] = useState(false)
+
+  return (
+    <section aria-label="Contact us." className={style.section}>
+      <h2>Contact Us</h2>
+      <p className={style.intro}>Any feedback? Please fill out our quick online inquiry form to send us a message.</p>
+      <div className={style.wrapper}>
+        {isSuccessful ? (
+          <p>Your message has been sent successfully!</p>
+        ) : (
+          <MessageForm setIsSuccessful={setIsSuccessful}></MessageForm>
+        )}
```

Comment:
Understand what you're doing here but worth considering whether it might be a better user experience if this wasn't a ternary. Yeah we should be showing a success message but should we be removing the form on success? Would it be reasonable to consider that someone might want to resubmit another message (or email for newsletter sign-up) once they have successfully submitted a response?

Reply:
I steal the idea from [paperhat](paperhat.ltd).

Reply:
Actually after reading the ACs I see that this is part of those so all good :grin: